lib/rspec/mocks/example_methods.rb

```diff
@@ -282,6 +282,17 @@ def have_received(method_name, &block)
         Matchers::HaveReceived.new(method_name, &block)
       end
 
+      # Turns off the verifying of partial doubles for the duration of the
+      # block, this is useful in situations where methods are defined at run
+      # time and you wish to define stubs for them but not turn off partial
+      # doubles for the entire run suite. (e.g. view specs in rspec-rails).
+      def without_verifying_partial_doubles
```

Comment:
Do you think `without_partial_double_verification` reads better for the method name? (I do...)